ckanext/stadtzhtheme/fanstatic/stadtzhtheme.css

```diff
@@ -273,3 +273,84 @@ ul.resource-list,
 .footer-links-right {
     float:right
 }
+
+.format-label {
+    background: url(/icons.png) no-repeat 0 0;
+}
+
+.format-label[data-format=wms], .format-label[data-format*=wms] {
+    background-position: -352px -62px;
+    width: 32px;
+    height: 35px;
```

Comment:
Wouldn't it be possible to define the `width` and `height` on `.format-label` level?

Reply:
Fixed.